wgpu-core/src/binding_model.rs

```diff
@@ -59,6 +59,8 @@ pub enum BindGroupLayoutEntryError {
 pub enum CreateBindGroupLayoutError {
     #[error(transparent)]
     Device(#[from] DeviceError),
+    #[error(transparent)]
+    Binding(#[from] BindingError),
```

Comment:
I added this here to avoid `match`-conversion of errors returned by the helper or introducing a new error type. But I can see a case that it makes more sense to preserve `BindingError` exclusively for bindings and not binding templates.